fsthttp/response.go

```diff
@@ -111,6 +111,10 @@ type ResponseWriter interface {
 	//
 	// To have an effect on the response, this must be called before any call to Write() or WriteHeader().
 	SetManualFramingMode(bool)
+
+	// Append a body onto the end of this response.
+	// This operation is performed in amortized constant time, and so should always be preferred to reading an entire body and then writing the same contents to another body.
```

Comment:
I think we need to make clear that the reader has to come from a Response's Body field, or make the function work with non-`HTTPBody` types [a la `Request.Send()`](https://github.com/fastly/compute-sdk-go/blob/main/fsthttp/request.go#L317-L333).

I might also make it more explicit that this is preferable to `io.Copy`, which I think most Go programmers will understand.  Something like, "This operation is performed in amortized constant time, and so should always be preferred to directly copying a body with io.Copy".